threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java

```diff
@@ -21,7 +23,7 @@ public class MessageSingleRecipientTest {
   @Test
   @Disabled
   public void sendMessage() {
```

Comment:
if you like to go an extra mile; we could not remove the @Disabled ... and implement this test easily with the new 'WebAppFixture' that Lukas introduced last week